framework/auth/__init__.py

```diff
@@ -117,20 +117,17 @@ def verify_two_factor(user_id, two_factor_code):
     return response
 
 
-def login(username, password):
+def login(email, password):
```

Comment:
You don't need to change it back, though; just wondering what the rationale was.